src/index.html

```diff
@@ -14,7 +14,7 @@
   <body>
     <!-- W taki sposbu dodajemy fragmenty w główny HTML-plik strony. -->
     <include src="./partials/example.html"></include>
-
+    <include src="./partials/footer.html"></include>
```

Comment:
Nie mogę wyświetlić Twojej sekcji, brakuje mi folderu z svg. Jeśli nie możesz go dodać to wrzuć te ikony do html, bez folderu zewnętrznego.